libraries/AP_Motors/AP_Motors_Class.cpp

```diff
@@ -19,6 +19,8 @@
 #include <GCS_MAVLink/GCS.h>
 #include <AP_Notify/AP_Notify.h>
 
+#define AP_MOTORS_SLEW_FILTER_CUTOFF 50.0f
```

Comment:
Where does this cutoff come from?  
From what I can tell this PR looks to address speed of response of propulsion systems, and that would be a function of the chosen propulsion system: e.g. ESC control strategy, motor peak torque, prop size and shape, etc.
Would this be better off being a parameter so we can adapt to different vehicle sizes?

Reply:
In the other PR its configurable. This was the value that gave the right kind of slew response I was looking for - so accurate but not oscillating. This only relates to the throttle slew, so is purely a function of how fast your fingers are.